dipy/reconst/dki.py

```diff
@@ -893,6 +950,158 @@ def axial_kurtosis(dki_params, min_kurtosis=0, max_kurtosis=3):
+
+    Parameters
+    ----------
+    ang : array (2,)
+        arrays containing the two polar angles
+    dt : array (6,)
+        elements of the diffusion tensor of the voxel.
+    MD : float
+        mean diffusivity of the voxel
+    kt : array (15,)
+        elements of the kurtosis tensor of the voxel.
+
+    Returns
+    -------
+    neg_kt : float
+        The inverse values of the directional kurtosis for the given direction.
+
+    Notes
+    -----
+    This function is used to refine the kurtosis maxima estimate
+
+    See also
+    --------
+    dipy.reconst.dki.kurtosis_maximum
+    """
+    n = np.array([sphere2cart(1, ang[0], ang[1])])
+    return -1. * _directional_kurtosis(dt, MD, kt, n)
+
+
+def _voxel_kurtosis_maximum(dt, MD, kt, sphere, gtol=1e-5):
+    """ Computes the maxima value of a single voxel kurtosis tensor
+
+    Parameters
+    ----------
+    dt : array (6,)
+        elements of the diffusion tensor of the voxel.
+    MD : float
+        mean diffusivity of the voxel
+    kt : array (15,)
+        elements of the kurtosis tensor of the voxel.
+    sphere : Sphere class instance, optional
+        The sphere providing sample directions for the initial search of the
+        maxima value of kurtosis.
+    gtol : float, optional
+        This input is to refine kurtosis maxima under the precision of the
+        directions sampled on the sphere class instance. The gradient of the
+        convergence procedure must be less than gtol before successful
+        termination. If gtol is None, fiber direction is directly taken from
+        the initial sampled directions of the given sphere object
+
+    Returns
+    -------
+    max_value : float
+        kurtosis tensor maxima value
+    max_dir : array (3,)
+        Cartesian coordinates of the direction of the maximal kurtosis value
+    """
+    # Estimation of maxima kurtosis candidates
+    AKC = _directional_kurtosis(dt, MD, kt, sphere.vertices)
+    max_val, ind = local_maxima(AKC, sphere.edges)
+    n = len(max_val)
+    max_dir = sphere.vertices[ind]
+
+    # Select the maxima from the candidates
+    max_value = max(max_val)
+    max_direction = max_dir[np.argmax(max_val.argmax)]
+
+    # refine maxima direction
+    if gtol is not None:
+        for p in range(n):
+            r, theta, phi = cart2sphere(max_dir[p, 0], max_dir[p, 1],
+                                        max_dir[p, 2])
+            ang = np.array([theta, phi])
+            ang[:] = opt.fmin_bfgs(_kt_maxima_converge, ang, args=(dt, MD, kt),
+                                   gtol=gtol, disp=False, retall=False)
+            k_dir = np.array([sphere2cart(1., ang[0], ang[1])])
+            k_val = _directional_kurtosis(dt, MD, kt, k_dir)
+            if k_val > max_value:
+                max_value = k_val
+                max_direction = k_dir
+
+    return max_value, max_direction
+
+
+def kurtosis_maximum(dki_params, sphere='symmetric362', gtol=1e-5,
```

Comment:
It occurs to me that there are places where this is spelled `maxima` and others where this is spelled `maximum`. How about simplifying and unifying, by spelling all of these as `max` ? For example, this function would be renamed `kurtosis_max`.